tox.ini

```diff
@@ -4,7 +4,7 @@ envlist =
     docs
     flake8
 skip_missing_interpreters = True
-
+isolated_build = True
```

Comment:
Wofür ist das?

Reply:
Für das Erstellen des Python Packages wird eine eigene virtuelle Umgebung verwendet.
Der default-name dieser virtuellen Umgebung ist `.package`.
Ob wir das aktuell brauchen, weiß ich aktuell leider noch nicht.
Ist aber praktisch, wenn wir andere bzw. zusätzliche built-tools wie [poetry](https://python-poetry.org/) oder [flit](https://flit.readthedocs.io/en/latest/) verwenden wollen in der Zukunft.
Die [tox-Doku](https://tox.readthedocs.io/en/latest/config.html#conf-isolated_build) kann es ggf. etwas besser erklären.

Reply:
können wir das auch im code in einem satz erklären? unterstützen .ini files kommentare?